src/routes/subscriptions.js

```diff
@@ -0,0 +1,59 @@
+const { StatusCodes } = require('http-status-codes');
+
+const express = require('express');
+const router = express.Router();
+
+// POST /
+// Recieve subscription notification
+router.post('/:id', (req, res) => {
+  const id = req.params.id;
+  res.sendStatus(StatusCodes.OK);
+  const subscription = getSubscription(id);
+  useSubscription(subscription);
+});
+
+router.put('/:id/:resource/:resourceId', (req, res) => {
+  const id = req.params.id;
+  res.sendStatus(StatusCodes.OK);
+  const subscription = getSubscription(id);
+  useSubscription(subscription, req.body);
+});
+function getSubscription(id) {
+  // dummy function for getting the subscription
+  // which can contain information to inform
+  // the app what to do when notified
+  console.log(id);
+  return {
+    subscriptionResource: {
+      resourceType: 'Subscription',
+      criteria: 'Patient?_id=1',
+      channel: {
+        type: 'rest-hook',
+        endpoint: 'http://example.com/subscription/1234',
+        header: ['content-type: application/fhir+json']
+      }
+    },
+    action: 'name'
+  };
+}
+function useSubscription(subscription, resource = null) {
```

Comment:
I wanted to keep the code more modular, I think it makes more sense to pass the subscription into this function, but I could see it both ways.  I did it this way because the `id` is not useful to the function beyond retrieving the subscription.